### PR TITLE
Add Vue.js Example + improve README

### PR DESCRIPTION
**Pull request by koddr**
* Add Vue.js Example (with single file component)
* Improve README

### CODE REVIEW

File: vuejs/package.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+  "name": "parcel-vuejs-example",
+  "version": "1.0.0",
+  "description": "Parcel Bundler Example with Vue.js",
+  "main": "src/main.js",
+  "scripts": {
```
Comment (brandon93s) [inline vuejs/package.json]: `start` and `build` won't work on windows. Could be fixed by using `cross-env`

Comment (koddr) [reply vuejs/package.json]: Okay, I fix it.

Comment (koddr) [reply vuejs/package.json]: Also, we're waiting for this issue https://github.com/lc60005457/parcel-plugin-vue/issues/5 for create more advanced Vue.js example (with images, who bugged now if you add in components).

**Comment by DeMoorJasper** [comment]
Parcel-plugin-vue is no longer recommended

**Pull request closed without merging**